README.md

```diff
@@ -12,7 +12,7 @@ wallet app.
 To compile the node, please install [cargo](https://doc.rust-lang.org/cargo/) and [rustup](https://rustup.rs/), then run the following commands:
 
     sudo apt update
-    sudo apt install -y build-essential pkg-config libzmq3-dev libssl1.0-dev libpq-dev
+    sudo apt install -y build-essential pkg-config libzmq3-dev libssl-dev libpq-dev cmake
```

Comment:
Afraid this will break back compatibility with Ubuntu. Can we have a separate Debian section? Or you did check that it still works on Ubuntu?

Reply:
Actually on my OS'es (Ubuntu 16.04 and 20.04) 
`sudo apt install -y build-essential pkg-config libzmq3-dev libssl-dev libpq-dev` worked
`sudo apt install -y build-essential pkg-config libzmq3-dev libssl1.0-dev libpq-dev`  did not work